prevent/lib/view/screens/notification/notificationConsul_screen.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:iconify_flutter/iconify_flutter.dart';
+import 'package:iconify_flutter/icons/zondicons.dart';
+import 'package:prevent/util/theme.dart';
+import 'package:prevent/view/screens/view_all_doctor/custom_search.dart';
+import 'package:prevent/view/widgets/home/side_bar.dart';
+
+class NotificationKonsulScreen extends StatelessWidget {
+  const NotificationKonsulScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Tabahin sidebar/drawernya

drawer: const SideBar(),
appbar: AppBar(
leading: Builder(
            builder: (BuildContext context) {
              return IconButton(
                padding: const EdgeInsets.only(left: 10),
                icon: const Iconify(
                  Ph.list,
                  size: 36,
                ),
                onPressed: () {
                  Scaffold.of(context).openDrawer();
                },
                tooltip: MaterialLocalizations.of(context).openAppDrawerTooltip,
              );
            },
          ),
)